creator/files/models.py

```diff
@@ -92,13 +93,19 @@ class Meta:
         related_name="files",
         help_text="The user who originally created this File",
     )
-
+    template_version = models.ForeignKey(
+        TemplateVersion,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="files",
+        help_text="The data template version this file conforms to",
```

Comment:
small thing, when this field is populated will we always know that the file conforms to the template_version?
otherwise, could say `The data template version this file should conform to`

Reply:
Actually, I'm not going to change it bc it would generate a new migration. Then I'd have to go through the annoying process of logging into my Dev EC2 and undoing the current migration and then redeploying